code/parachain/frame/staking-rewards/src/test/test_update_reward_pools.rs

```diff
@@ -217,6 +218,7 @@ fn update_accumulates_properly() {
 				owner: CHARLIE,
 				fnft_collection_id: STAKING_FNFT_COLLECTION_ID,
 				fnft_instance_id: stake_id,
+				claimed_amount: BTreeMap::from([(USDT::ID, USDT::units(1) / 1000 * 6)]),
```

Comment:
prefer to use `[..].into_iter().collect()` over `BTreeMap::from([..])`

Reply:
There is also a `btree_map` helper function that can be used as well, but .iter.collect tends to format more nicely